test/Feature/AuthSpec.hs

```diff
@@ -18,50 +18,11 @@ spec = beforeAll
   it "hides tables that anonymous does not own" $
     get "/authors_only" `shouldRespondWith` 404
 
-  it "indicates login failure (BasicAuth)" $ do
-    let auth = authHeaderBasic "postgrest_test_author" "fakefake"
-    request methodGet "/authors_only" [auth] ""
-      `shouldRespondWith` 401
-
-  it "allows users with permissions to see their tables (BasicAuth)" $ do
```

Comment:
Maybe add these role permissions tests back in but instead of sending basic auth send along a JWT.

(But not tests for login failure since valid JWTs are always trusted)